Description: install files in correct locations
Index: tempo/src/tempo.f
===================================================================
--- tempo.orig/src/tempo.f
+++ tempo/src/tempo.f
@@ -188,7 +188,7 @@ c  Get command-line arguments
 
 c  Parse tempo.cfg file
 
- 	cfgpath=path(1:lpth)//'/tempo.cfg'
+ 	cfgpath='/etc/tempo.cfg'
         call cfgin(cfgpath,ut1file,obsyfile,tdbfile)  
 
 c  Open primary output file (tempo.lis)
Index: tempo/src/tparin.f
===================================================================
--- tempo.orig/src/tparin.f
+++ tempo/src/tparin.f
@@ -93,13 +93,13 @@ c     default values of parameters
 
       infile = ''	
 
-      call getenv ('TEMPO',path)
+      path = '/usr/share/tempo/'
       lpth = index(path,' ')-1
       if (lpth.eq.0) then
         print *, "You must set the $TEMPO environment variable in order to run tempo."
         stop
       endif
-      hlpfile = path(1:lpth)//'/tempo.hlp'	
+      hlpfile = '/usr/share/tempo/tempo.hlp'	
 
 
       narg = iargc()
@@ -129,7 +129,7 @@ c     default values of parameters
               path = getparm(s,i,ii,iarg,narg)
               if (path.eq.'') goto 9999 ! error
               lpth = index(path,' ')-1
-              hlpfile = path(1:lpth)//'/tempo.hlp'
+              hlpfile = '/usr/share/tempo/tempo.hlp'
 	    else if (s(i:i).eq.'D') then
 	      ldesign = .true.
 	      open(37,file='design.tmp',form='unformatted',
